syntax = "proto3";

service MicroService{
    rpc SendString(str) returns (ResponseInt) {}
    rpc SendInt(number) returns (ResponseString) {}
}

message str{
    string item = 1;
}

message number{
    int32 num = 1;
}

message ResponseString{
    bool status = 1;
    string response = 2;
}

message ResponseInt{
    bool status = 1;
    int32 response = 2;
}
